src/main/java/org/example/P04_TheOutputFor.java

```diff
@@ -0,0 +1,8 @@
+package org.example;
+/*What is the output for System.out.println(-6+4*2)*/
+public class P04_TheOutputFor {
```

Comment:
Rename the class name to a proper name. P01_OperationsPriority. Start from P01 so it will be easy to track with udemy course. It doesn't matter that you have two problems starting with P01